Description: Proper YAML syntax for rootservers file
 The default YAML parser in Ruby 1.9 is more strict with the format. This patch
 converts the rootservers file to a more correct (and more readable) format.
Author: Antonio Terceiro <terceiro@debian.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=676173

--- zonecheck-3.0.3.orig/etc/zonecheck/rootservers
+++ zonecheck-3.0.3/etc/zonecheck/rootservers
@@ -11,21 +11,44 @@
 # This list can be generated by the following shell-script (sh):
 #
 #  for ns in `dig +short . ns | tr 'A-Z' 'a-z' | sort` ; do
-#    ips=`(dig +short $ns a; dig +short $ns aaaa) | tr '\n' ',' | sed 's/,$//'`
-#    echo "$ns: [ $ips ]"
+#    echo "$ns:"
+#    for ip in $(dig +short $ns a; dig +short $ns aaaa); do
+#      echo "  - '$ip'"
+#    done
 #  done
-# TODO: bad syntax?
 
-a.root-servers.net.: [ 198.41.0.4 , 2001:503:ba3e::2:30 ]
-b.root-servers.net.: [ 192.228.79.201 ]
-c.root-servers.net.: [ 192.33.4.12 ]
-d.root-servers.net.: [ 128.8.10.90 ]
-e.root-servers.net.: [ 192.203.230.10 ]
-f.root-servers.net.: [ 192.5.5.241 , 2001:500:2f::f ]
-g.root-servers.net.: [ 192.112.36.4 ]
-h.root-servers.net.: [ 128.63.2.53 , 2001:500:1::803f:235 ]
-i.root-servers.net.: [ 192.36.148.17, 2001:7FE:0:0:0:0:0:53 ]
-j.root-servers.net.: [ 192.58.128.30 , 2001:503:c27::2:30 ]
-k.root-servers.net.: [ 193.0.14.129 , 2001:7fd::1 ]
-l.root-servers.net.: [ 199.7.83.42 , 2001:500:3::42 ]
-m.root-servers.net.: [ 202.12.27.33 , 2001:dc3::35 ]
+a.root-servers.net.:
+  - '198.41.0.4'
+  - '2001:503:ba3e::2:30'
+b.root-servers.net.:
+  - '192.228.79.201'
+c.root-servers.net.:
+  - '192.33.4.12'
+d.root-servers.net.:
+  - '128.8.10.90'
+  - '2001:500:2d::d'
+e.root-servers.net.:
+  - '192.203.230.10'
+f.root-servers.net.:
+  - '192.5.5.241'
+  - '2001:500:2f::f'
+g.root-servers.net.:
+  - '192.112.36.4'
+h.root-servers.net.:
+  - '128.63.2.53'
+  - '2001:500:1::803f:235'
+i.root-servers.net.:
+  - '192.36.148.17'
+  - '2001:7fe::53'
+j.root-servers.net.:
+  - '192.58.128.30'
+  - '2001:503:c27::2:30'
+k.root-servers.net.:
+  - '193.0.14.129'
+  - '2001:7fd::1'
+l.root-servers.net.:
+  - '199.7.83.42'
+  - '2001:500:3::42'
+m.root-servers.net.:
+  - '202.12.27.33'
+  - '2001:dc3::35'
